server/aesdsocket.c

```diff
@@ -0,0 +1,216 @@
+#include <stdio.h>
```

Comment:
@maheshchnt you shouldn't check the aesdsocket file into your repository, this can cause problems with your buildroot build/install.